src/python/grpcio_health_checking/setup.py

```diff
@@ -46,12 +46,12 @@
 }
 
 SETUP_REQUIRES = (
-    'grpcio-tools>=0.15.0',
+    'grpcio-tools>={version}'.format(version=grpc_version.VERSION),
 )
 
 INSTALL_REQUIRES = (
     'protobuf>=3.0.0',
-    'grpcio>=0.15.0',
+    'grpcio>={version}'.format(version=grpc_version.VERSION)',
```

Comment:
Yes, I'm running the tests locally this time and that typically takes 10+ minutes.